<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Homepage</class>
 <widget class="QWidget" name="Homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="randomGameLbl">
    <property name="geometry">
      <rect>
        <x>10</x>
        <y>80</y>
        <width>50</width>
        <height>40</height>
      </rect>
    </property>
    <property name="text">
     <string>Random Game: </string>
    </property>
  </widget>
  <widget class="QPushButton" name="playRandom">
    <property name="geometry">
      <rect>
        <x>10</x>
        <y>100</y>
        <width>50</width>
        <height>18</height>
      </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
  </widget>
  <widget class="QPushButton" name="newRandom">
    <property name="geometry">
      <rect>
        <x>65</x>
        <y>100</y>
        <width>18</width>
        <height>18</height>
      </rect>
    </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
